ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ContinuousHikingStateMachine/ReadyToPlanState.java

```diff
@@ -65,11 +60,13 @@ public void doAction(double timeInState)
 
       if (continuousHikingParameters.getStepPublisherEnabled())
       {
-         continuousPlanner.getImminentStanceFromLatestStatus(latestFootstepStatusMessage, controllerFootstepQueueMonitor.getControllerFootstepQueue());
+         if (controllerFootstepQueueMonitor.getFootstepStatusMessage() != null)
```

Comment:
This can be a single if statement, just use &&

Reply:
Done